samples/XCT.Sample/ViewModels/Markup/SearchViewModel.cs

```diff
@@ -77,18 +80,18 @@ public SearchViewModel()
 
 		public class Tweet : BaseViewModel
 		{
-			public string AuthorImage { get; set; }
+			public string AuthorImage { get; set; } = string.Empty;
 
-			public string Header { get; set; }
+			public string Header { get; set; } = string.Empty;
 
-			public List<TextFragment> Body { get; set; }
+			public List<TextFragment> Body { get; set; } = Enumerable.Empty<TextFragment>().ToList();
```

Comment:
nit: Why not `new()`?

Reply:
No reason; I had just gotten into the groove of using `Enumerable.Empty` in other places.  I'll switch it to `new()`

Reply:
I think it's only needed for Enumerables. There is separate method for arrays, and for other collections you can just use their constructors.

Also, since you updated us to C# 9, you can literally use `new()` here:
`public List<TextFragment> Body { get; set; } = new();`

Love this new syntax =)